### DISCUSSION TITLE
How to limit column base on number

### DISCUSSION THREAD

**Discussion by neroshin** [iddan/react-spreadsheet#291] [Ideas]
Hi support , can you help me how can I limit the column into 7 column , including limit to 7 solumn when I copy from another speadsheet that has 10 column then paste to another sheet , the particular 10 column should reduce to 7 column after paste , can you help to that part?

**Comment by iddan** [comment]
This is not currently possible, but PRs are welcome! Currently, I don't actively develop new features myself but I would love to review a PR.
